far/viewer.hpp

```diff
@@ -306,6 +307,9 @@ class Viewer:public SimpleScreenObject
 	std::vector<wchar_t> ReadBuffer;
 	F8CP f8cps;
 	std::optional<bool> m_GotoHex;
+	int m_PrevXX2;
+	std::size_t m_BytesPerLine;
+	const std::wstring m_HexBorderLine;
```

Comment:
We generally don't prefix `size_t` and `std::wstring` is aliased as `string`.

Reply:
Modified as requested